server/db/schema.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/cushion');
+
+var TechieSchema = new mongoose.Schema({
+            userName: {type: String, unique: true},
+            password: String,
+            Name: String,
+            email: String,
```

Comment:
We should be consistent with either lower or uppercase for the start of all the fields (we don't need to camelcase, i.e. userName can just be username